src/quo2/components/buttons/button/view.cljs

```diff
@@ -29,10 +29,11 @@
    only icon
    [button {:icon-only? true} :i/close-circle]"
   [_ _]
-  (let [pressed? (reagent/atom false)]
+  (let [pressed-state? (reagent/atom false)]
```

Comment:
Should the default value be set by pressed?
We can always update later if we find issues with it anyway